src/eduid_idp/login.py

```diff
@@ -78,9 +79,17 @@ def perform_login(self, ticket: SSOLoginData) -> bytes:
         binding_out = resp_args.get('binding_out')
         destination = resp_args.get('destination')
 
+        if self.session:
+            self.session['user_eppn'] = user.eppn
+            self.session.commit()
+
         check_for_pending_actions(self.context, user, ticket, self.sso_session)
         # We won't get here until the user has completed all login actions
 
+        if self.session:
+            self.session['is_logged_in'] = True
```

Comment:
Didn't we say that this was the authn service responsibility?

Reply:
You're right - we said we should let authn continue setting that flag for now. My bad.